Add details about deprecated runtimes (#3595)
astronomer/docs: astro/upgrade-runtime.md

:::

1. Contact your Astronomer account team and request the ability to run deprecated versions of Astro Runtime.
2. Create a [Personal Access Token](https://cloud.astronomer.io/token)(PAT) to use for authentication to Astro.

>     :::note

Reviewer: ```suggestion
 
       :::info
```

>     You only need a PAT for the first Deployment in your Organization running a deprecated version of the Runtime.

Reviewer: ```suggestion
    You only need a PAT to create the first Deployment in your Organization running a deprecated version of Astro Runtime. You can then create all subsequent Deployments using either a Workspace or Organization API token.
```
Couldn't hurt to clarify this here as well

>     :::

Reviewer: ```suggestion
    :::
    
```

> 3. Create a Deployment using the [Astro API](https://docs.astronomer.io/api/platform-api-reference/deployment/create-deployment). In your request, specify your deprecated Astro Runtime version using the `astroRuntimeVersions` property.

Reviewer: ```suggestion
3. Create a Deployment using the [Astro API](https://docs.astronomer.io/api/platform-api-reference/deployment/create-deployment). In your request, specify your PAT in the authorization header, and your deprecated Astro Runtime version using the `astroRuntimeVersions` property.
```

> After you create the Deployment, you can manage the Deployment using any interface, including the Astro CLI and [Astro UI](deployment-settings.md). You can also create additional Deployments with deprecated Runtimes using a [Deployment API Token](deployment-api-tokens.md).

Reviewer: ```suggestion
After you create the Deployment, you can manage the Deployment using any interface, including the Astro CLI and [Astro UI](deployment-settings.md). You can also create additional Deployments with deprecated Runtimes using the Astro API and either a Workspace or Organization API token.
```

:::

1. Contact your Astronomer account team and request the ability to run deprecated versions of Astro Runtime.
2. Create a [Personal Access Token](https://cloud.astronomer.io/token)(PAT) to use for authentication to Astro.
 
       :::info

>     You only need a PAT to create the first Deployment in your Organization running a deprecated version of Astro Runtime. You can then create all subsequent Deployments using either a Workspace or Organization API token.

Author: ```suggestion
    You only need a PAT to create the first Deployment in your Organization running a deprecated version of Astro Runtime. You can then create all subsequent Deployments using either a Workspace or Organization API token with permissions to create a Deployment.
```
